config.go

```diff
@@ -53,6 +53,7 @@ var (
 		"image":              hclspec.NewAttr("image", "string", true),
 		"init":               hclspec.NewAttr("init", "bool", false),
 		"init_path":          hclspec.NewAttr("init_path", "string", false),
+		"labels":             hclspec.NewAttr("labels", "list(map(string))", false),
```

Comment:
Any reason why it's not just a map(string) ?

Reply:
i did look that up from the docker driver... i was not sure as well, never used hcl before

Reply:
https://github.com/hashicorp/nomad/blob/bdf2555b385f547cbf613d1636eac103a1c58ce8/drivers/docker/config.go#L361

Reply:
IC. But the docker config example shows no "[" , "]" list elements, just a map. Can you try to run you code against a simplified hcl without the brackets?

Reply:
sure, works as well